.travis.yml

```diff
@@ -130,13 +130,22 @@ before_script:
 
 script:
   - |
+    BUILD_EXIT_CODE=0
     for meta in "${BUILD_WHEEL_METAS[@]}"; do
         _f=${meta#wheels/} ; wheel=${_f%%/*}
         echo "Building '$wheel' wheel from config: $meta"
-        $STARFORGE_CMD --debug wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"
-        echo "Testing '$wheel' wheel"
-        $STARFORGE_CMD --debug test_wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"
+        $STARFORGE_CMD --debug wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"; STARFORGE_EXIT_CODE=$?
+        if [ "$STARFORGE_EXIT_CODE" -eq 0 ]; then
+            echo "Testing '$wheel' wheel"
+            $STARFORGE_CMD --debug test_wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"
+        elif [ "$STARFORGE_EXIT_CODE" -eq 1 ]; then
+            echo "Building '$wheel' wheel failed"
+            BUILD_EXIT_CODE=1
+        else
+            echo "`starforge wheel` exited with code '$STARFORGE_EXIT_CODE', skipping wheel test"
```

Comment:
Whoops, forgot to escape these backticks. But it's not failing the builds, so I'll let it finish running?